src/docs/tutorials/01-getting-started-with-ftw/index.md

```diff
@@ -22,10 +22,10 @@ application requires varying levels of development knowledge. For this
 tutorial it is enough to know how to open a terminal to run some
 commands and edit text files to change configuration values.
 
-For more information on what skills are needed for customization, see
-the
-[What development skills are needed?](/background/development-skills/)
-article.
+The source code for FTW is available in [the FTW Github
```

Comment:
```suggestion
The source code for FTW is available at [the FTW GitHub
```

Reply:
@kpuputti or should we just put:

The FTW application is available in [GitHub](link to repo).

Reply:
At least `s/in/at/`.

How about:

FTW is available at the [sharetribe/flex-template-web](...) GitHub repository.

Your call.

Reply:
That final one looks pretty lean, let's go with that.